lib/evmone/baseline.cpp

```diff
@@ -136,6 +139,19 @@ template <>
     return instr_fn(state, pos);
 }
 
+/// A helper to invoke the instruction implementation of the given opcode Op.
+template <evmc_opcode Op>
+[[gnu::always_inline]] inline code_iterator invoke(
```

Comment:
Still I wonder if it's simpler to inline this helper manually into the macro.

Reply:
I want to have the macro as small as possible: it is only needed for `goto exit`.